feat(self-hosted): Add experimental page (#11473)
getsentry/sentry-docs: develop-docs/self-hosted/experimental/reverse-proxy.mdx

sidebar_order: 80
---

<Alert title="Important" level="warning">

>     The docs below are contributed by the community. Sentry does not officially provide support for setting up reverse proxies for your self-hosted Sentry setup. Follow these guidelines at your own risk as these docs are not maintained nor kept up to date!

Reviewer: I think it'd be better to keep this banner generic so we can copy paste this to every file under this `experimental` section. (Can you do that as well?)

```suggestion
    These are community-contributed docs. Sentry does not officially provide support for self-hosted configurations beyond the defaults.
```

Author: CSP docs are written by us but are technically experimental as well so I don't think we should necessarily have generic banners for every file under the `experimental` section. I'm on board with the same generic banner for community contributed docs though
